nservicebus/upgrades/azure-functions-service-bus-in-process-2to3.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Azure Functions (in-process) for Service Bus Upgrade Version 2 to 3
+summary: How to upgrade Azure Functions (in-process) with Azure Service Bus from version 2 to 3
+component: ASBFunctions
+reviewed: 2021-12-16
+related:
+ - nservicebus/upgrades/7to8
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 8
```

Comment:
Functions versions 2 and 3 are both NServiceBus Core 7, right?
```suggestion
 - 7
```

Reply:
In process v3 targets core v8 https://github.com/Particular/NServiceBus.AzureFunctions.InProcess.ServiceBus/blob/master/src/NServiceBus.AzureFunctions.InProcess.ServiceBus/NServiceBus.AzureFunctions.InProcess.ServiceBus.csproj#L18